### PR TITLE
Use better variable names

### PR DESCRIPTION
**Pull request by korikuzma**
- normalize_endpoint --> endpoint_name (validators)
- normalize_endpoint --> try_longest_compatible (mane transcript) 

I believe this will help with code-reuse in #251

### CODE REVIEW

File: variation/__init__.py [hunk 1]
```diff
@@ -23,6 +23,17 @@
...
 
+logging.getLogger("boto3").setLevel(logging.INFO)
+logging.getLogger("botocore").setLevel(logging.INFO)
+logging.getLogger("urllib3").setLevel(logging.INFO)
+logging.getLogger("python_jsonschema_objects").setLevel(logging.INFO)
+logging.getLogger("hgvs.parser").setLevel(logging.INFO)
+logging.getLogger("biocommons.seqrepo.seqaliasdb.seqaliasdb").setLevel(logging.INFO)
+logging.getLogger("biocommons.seqrepo.fastadir.fastadir").setLevel(logging.INFO)
+
+# logging.getLogger("bioregistry.resource_manager").setLevel(logging.INFO)
```
Comment (jsstevenson) [inline variation/__init__.py:34]: You probably don't need this, right?

File: variation/schemas/schemas.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+"""Module for general schemas used throughout the app"""
```
Comment (jsstevenson) [inline variation/schemas/schemas.py:1]: maybe a more descriptive filename (metaschemas? idk)

File: variation/schemas/schemas.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+from enum import Enum
+
+
+class Endpoint(str, Enum):
+    """Define contrains for endpoint names"""
+
+    TO_VRS = "toVRS"
```
Comment (jsstevenson) [inline variation/schemas/schemas.py:10]: any reason why this isn't snake case?

**Review by jsstevenson — CHANGES_REQUESTED** [review]

File: variation/schemas/schemas.py (hunk 3 above)
Comment (korikuzma) [reply variation/schemas/schemas.py:10]: I was just going with the actual endpoints that we use

File: variation/__init__.py (hunk 1 above)
Comment (korikuzma) [reply variation/__init__.py:34]: Lol was hoping you'd miss that part. No I don't. I just didn't want to make another commit, but will remove it.

File: variation/schemas/schemas.py (hunk 2 above)
Comment (korikuzma) [reply variation/schemas/schemas.py:1]: When with app_schemas

File: variation/__init__.py (hunk 1 above)
Comment (jsstevenson) [reply variation/__init__.py:34]: FWIW you can just squash commits on merge ("squash" is a v satisfying word)

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**